package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>rosjava_core</name>
-  <version>0.3.5</version>
```

Comment:
apologies, that was my first pull request. I thought that commit would only stay in my fork, so I could use the repo in case pull request wouldn't be accepted. I reverted all commits afterwards with

push -f origin 6a90988f6b06116bef34caf5bdafe1712bb0480c:kinetic

Then I committed a change that removed the comment with wrong white space